freeipa/src/main/java/com/sequenceiq/freeipa/util/ASNEncoder.java

```diff
@@ -0,0 +1,77 @@
+package com.sequenceiq.freeipa.util;
+
+import com.cloudera.thunderhead.service.usermanagement.UserManagementProto.ActorKerberosKey;
+
+import java.util.Base64;
+import java.util.List;
+
+import org.bouncycastle.asn1.ASN1Encodable;
+import org.bouncycastle.asn1.DERInteger;
+import org.bouncycastle.asn1.DEROctetString;
+import org.bouncycastle.asn1.DERSequence;
+import org.bouncycastle.asn1.DERTaggedObject;
+
+public final class ASNEncoder {
+
+    private static final int TAG_0 = 0;
+
+    private static final int TAG_1 = 1;
+
+    private static final int TAG_2 = 2;
+
+    private static final int TAG_3 = 3;
+
+    private static final int TAG_4 = 4;
```

Comment:
do these constants really add much value? If you want the code to be more descriptive, perhaps a small utility method like:
```
static DERTaggedObject tag(int tag, ASN1Encodable obj) {
  return new DERTaggedObject(true, tag, obj);
}
```

would do the trick?

Reply:
This way I cheat checkstyle.